### PR TITLE
chore: surface CPR docs, and reformat docstrings

### PR DESCRIPTION
**Pull request by kthytang**
Staged docs - https://cloud.devsite.corp.google.com/python/docs/reference/aiplatform/latest/google.cloud.aiplatform.prediction

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-aiplatform/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: google/cloud/aiplatform/prediction/local_model.py [hunk 1]
```diff
@@ -178,28 +184,34 @@ def build_cpr_model(
...
+            container_workdir/
+            |-- predictor.py
+            |-- requirements.txt
+            |-- user_code/
+            |   |-- utils.py
+            |   |-- custom_package.tar.gz
+            |   |-- ...
+            |-- ...
+
+        Too exclude files and directories from being copied into the built container images, create a
```
Comment (abcdefgs0324) [inline google/cloud/aiplatform/prediction/local_model.py:200]: To

File: google/cloud/aiplatform/prediction/local_model.py [hunk 2]
```diff
@@ -353,34 +372,34 @@ def deploy_to_local_endpoint(
...
-                Optional. The port on the host that the port, AIP_HTTP_PORT, inside the container
+                Optional. The port on the host that the port, ``AIP_HTTP_PORT``, inside the container
                 will be exposed as. If it's unset, a random host port will be assigned.
             gpu_count (int):
                 Optional. Number of devices to request. Set to -1 to request all available devices.
-                To use GPU, set either `gpu_count` or `gpu_device_ids`.
-                The default value is -1 if gpu_capabilities is set but both of gpu_count and
-                gpu_device_ids are not set.
+                To use GPU, set either ``gpu_count`` or ``gpu_device_ids``.
+                The default value is -1 if ``gpu_capabilities`` is set but both of ``gpu_count`` and
```
Comment (abcdefgs0324) [inline google/cloud/aiplatform/prediction/local_model.py:390]: both of -> both

**Review by abcdefgs0324 — APPROVED** [review]
Some minor comments.

File: google/cloud/aiplatform/prediction/local_endpoint.py [hunk 3]
```diff
@@ -376,7 +376,7 @@ def predict(
         """
         if self.container_is_running is False:
             raise RuntimeError(
-                "The local endpoint is not serving traffic. Please call `serve()`."
+                "The local endpoint is not serving traffic. Please call ``serve()``."
```
Comment (abcdefgs0324) [inline google/cloud/aiplatform/prediction/local_endpoint.py:379]: You may need to update the unit tests if you update the code.

Comment (kthytang) [reply google/cloud/aiplatform/prediction/local_endpoint.py]: Good catch! Didn't mean to modify this.

File: google/cloud/aiplatform/prediction/local_model.py (hunk 2 above)
Comment (kthytang) [reply google/cloud/aiplatform/prediction/local_model.py]: Done.

File: google/cloud/aiplatform/prediction/local_model.py (hunk 1 above)
Comment (kthytang) [reply google/cloud/aiplatform/prediction/local_model.py]: Thanks! Done.

File: google/cloud/aiplatform/prediction/local_model.py [hunk 4]
```diff
@@ -102,8 +102,8 @@ def __init__(
                 the network.
 
         Raises:
-            ValueError: If `serving_container_spec` is specified but `serving_container_spec.image_uri`
-                is None. Also if `serving_container_spec` is None but `serving_container_image_uri` is
+            ValueError: If ``serving_container_spec`` is specified but ``serving_container_spec.image_uri``
+                is None. Also if ``serving_container_spec`` is None but ``serving_container_image_uri`` is
                 None.
```
Comment (dandhlee) [inline google/cloud/aiplatform/prediction/local_model.py:107]: nit:

```suggestion
            ValueError: If ``serving_container_spec`` is specified but ``serving_container_spec.image_uri``
                is ``None``. Also if ``serving_container_spec`` is None but ``serving_container_image_uri`` is
                ``None``.
```

File: google/cloud/aiplatform/prediction/local_model.py [hunk 5]
```diff
@@ -240,7 +253,7 @@ def build_cpr_model(
             local model: Instantiated representation of the local model.
 
         Raises:
-            ValueError: If handler is None or if handler is PredictionHandler but predictor is None.
+            ValueError: If handler is None or if handler is ``PredictionHandler`` but predictor is None.
```
Comment (dandhlee) [inline google/cloud/aiplatform/prediction/local_model.py:256]:
```suggestion
            ValueError: If handler is ``None`` or if handler is ``PredictionHandler`` but predictor is ``None``.
```

**Review by dandhlee — APPROVED** [review]
LGTM! Thank you for working through the weird spacing issues 😓

**Review by rosiezou — APPROVED** [review]
LGTM

**Pull request merged**